src/borg/archiver.py

```diff
@@ -76,10 +76,12 @@ def wrapper(self, args, **kwargs):
             if argument(args, fake):
                 return method(self, args, repository=None, **kwargs)
             elif location.proto == 'ssh':
-                repository = RemoteRepository(location, create=create, lock_wait=self.lock_wait, lock=lock, args=args)
+                repository = RemoteRepository(location, create=create, lock_wait=self.lock_wait, lock=lock,
+                                              append_only=getattr(args, 'append_only', False), args=args)
             else:
                 repository = Repository(location.path, create=create, exclusive=argument(args, exclusive),
-                                        lock_wait=self.lock_wait, lock=lock)
+                                        lock_wait=self.lock_wait, lock=lock,
+                                        append_only=getattr(args, 'append_only', False))
```

Comment:
maybe move append_only=getattr(...) to before the if-block so you don't need to duplicate it.